example/src/App.tsx

```diff
@@ -78,31 +67,11 @@ const App: React.FC<Props> = () => {
             component={WebviewScreen}
             options={{ title: 'Turbo Native Demo' }}
           />
-          <Stack.Screen
-            name={Routes.One}
-            component={WebviewScreen}
-            options={{ title: "How'd You Get Here?" }}
-          />
```

Comment:
Maybe we could add few words in the docs saying that you can manually add a screen to navigator to set its initial title or presentation?